src/routes/discord.js

```diff
@@ -38,14 +38,14 @@ router.get('/callback', catchAsyncErrors(async (req, res) => {
         //const client = DiscordClient.instance;
```

Comment:
```suggestion
```

Reply:
I deleted them in my fork, but I figured I would let Versx make that call or keep them there for future use.